viper/opcodes.py

```diff
@@ -71,11 +71,11 @@
 }
 
 pseudo_opcodes = {
-    'CLAMP': [None, 3, 1, 45],
+    'CLAMP': [None, 3, 1, 45 + 25],
```

Comment:
Why not just change it to 70?

Reply:
Sure will do so, I just kept it separate at the time to know the delta easily. Forgot to change it to a single value hehe ;)